let CustomErorHandler,User;_991‍.x([["default",()=>_991‍.o]]);_991‍.w("../../services/CustomErrorHandler",[["default",["CustomErorHandler"],function(v){CustomErorHandler=v}]]);_991‍.w("../../models",[["User",["User"],function(v){User=v}]]);


const userController = {

   async me(req,res,next) {

    try{

        const user = await User.findOne({_id : req.user._id}).select('-__v -password -updatedAt ');
        if(!user) {
            return next(CustomErorHandler.notFound());
        }
        res.json(user);

    } catch(err) {
        return next(err);
    }

    }
}

_991‍.d(userController);